Sugarcoat resources

ShivanKaul opened the pull request.

ShivanKaul commented on metadata.json:
```diff
@@ -1,8 +1,26 @@
 [
-{
-    "name": "brave-fix",
-    "aliases": ["bf"],
```
Do we need this alias?

antonok-edm replied:
@ryanbr

antonok-edm reviewed (approved):
Looks good to me but let's confirm with @ryanbr about the `bf` alias before merging

ryanbr commented:
Yeah we still need "bf" @ShivanKaul

ShivanKaul commented:
@ryanbr gotcha, added that back in. It would be nice if we could generate the alias automatically; I don't want to retrofit an alias for every sugarcoat resource just because of brave-fix though, so leaving it as a manual change, added a note to `generateMetadataJsonFromScriptResources.js` that a manual fixup is needed after running it.
